### DISCUSSION TITLE
Preferred method for donations

### DISCUSSION THREAD

**Discussion by ratty222** [TechbeeAT/jtxBoard#6] [General]
Is there prefered avenue for donations? I haven't renewed my liberapay contribution that expired a few weeks ago, but I read that one of the reasons to shift to github was ease of donations. I'd be happy to sponsor you here.

**Comment by patrickunterwegs** [comment]
Hey Brent :-)
Thanks for your message! 
Yes, donations was one of the reasons, in Github there are those integrated ways for getting financial support that don't exist in Gitlab (on the project start-page here on Github you can see that section on the right side). But I actually don't have experience with them. There is no preferred way for donations, but I liked the idea of ko-fi that claims to not take any commission. So feel free to choose 🙂 

And thank you very much for any past and future support 🙂

**Reply by ratty222** [reply]
You are very welcome. You have brought us a really amazing app that goes beyond any to-do list I've ever used.  I didn't even know VJournal was a thing, and now I would struggle to function without it.
